docs/sdk/python/complementary-examples.md

```diff
@@ -312,17 +314,27 @@ set_network(Testnet)
 client = SolarClient('https://sxp.testnet.sh/api')
 
 # Step 1: Retrieve the incremental nonce of the sender wallet
-senderWallet = client.wallets.get('YOUR_SENDER_WALLET_ADDRESS')
+senderWallet = client.wallets.get('SENDER_WALLET_ADDRESS')
 nonce = int(senderWallet['data']['nonce']) + 1
 
 # Step 2: Create the transaction
-transaction = MultiPayment()
-
+transaction = MultiSignatureRegistration()
 transaction.set_type_group(TRANSACTION_TYPE_GROUP.CORE)
 transaction.set_nonce(nonce)
-transaction.add_payment(1, 'D6Z26L69gdk9qYmTv5uzk3uGepigtHY4ax')
-transaction.add_payment(2, 'DNjuJEDQkhrJ7cA9FZ2iVXt5anYiM8Jtc9')
-transaction.schnorr_sign('this is a top secret passphrase')
+transaction.set_sender_public_key('SENDER_WALLET_PUBLIC_KEY')
+transaction.set_min(2)
+transaction.set_public_keys([
+    'participant_1_pk',
+    'participant_2_pk'
+])
+transaction.add_participant(
+    'participant_3_pk'
+)
+transaction.multi_sign('participant_1_passphrase', 0)
+transaction.multi_sign('participant_2_passphrase', 1)
+transaction.multi_sign('participant_3_passphrase', 2)
```

Comment:
Removed those statements in [f283a61](https://github.com/Solar-network/docs/pull/12/commits/f283a61dca517a35eccee19cb67a2adaaa376d36).

Regarding possible update needed in Crypto API reference; are you referring to ` def multi_sign(self, passphrase, index):` in `transactions.builder.base.BaseTransactionBuilder`? I failed to find any other reference.

Reply:
Yeah, so because it _is_ under the builder base there's no need to change anything regarding this in the API ref. 

If it _were_ under the MultiSignature Reg. ref then it would have needed changed as well, but that obviously doesn't apply.